webplugin/js/app/zendesk-chat-service.js

```
@@ -74,214 +71,229 @@ function ZendeskChatService() {
-                    console.log("File has been sent successfully : ", data);
+                    console.log('File has been sent successfully : ', data);
                 }
             });
         }
     };
 
     _this.handleBotMessage = function (event) {
-        console.log("handleBotMessage: ", event);
-        if (event.message.metadata.hasOwnProperty("KM_ASSIGN_TO")) {
+        console.log('handleBotMessage: ', event);
+        if (event.message.metadata.hasOwnProperty('KM_ASSIGN_TO')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)